html/pfappserver/lib/pfappserver/Form/Violation.pm

```diff
@@ -320,6 +337,34 @@ sub options_template {
     return @templates;
 }
 
+=head2 options_durations
+
+Populate the access duration select field with the available values defined
+in the pf.conf configuration file.
+
+=cut
+
+sub options_durations {
```

Comment:
Instead of copy/pasting the subroutine, it would be better to reuse the existing one in the other form or create a util function called by both forms